cfgov/v1/tests/management/commands/test_update_data_snapshot_values.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import call_command
+from django.test import Client, TestCase
+
+from wagtail.wagtailcore.blocks import StreamValue
+
+from scripts import _atomic_helpers as atomic
+from v1.models.browse_page import BrowsePage
+from v1.tests.wagtail_pages.helpers import publish_page
+
+django_client = Client()
```

Comment:
I may be missing something, but `django_client` doesn't appear to be used anywhere, so I'm removing it since it has `self.client`.